general/releases.rst

```diff
@@ -9,6 +9,26 @@ Cadence
 *******
 New versions of OP-TEE are released four times a year, i.e., quarterly releases.
 
+.. _release_dates:
+
+Release dates
+=============
+Starting from version 3.10.0 we track the old and also show the releases being
+planned for the future in the table below. The dates will tell whether it is an
+old, upcoming or future release.
+
++---------------+------------+--------------+
+| Version       | Start date | Release date |
++===============+============+==============+
+| OP-TEE 3.13.0 | 16/Jan/21  | 16/Apr/21    |
++---------------+------------+--------------+
+| OP-TEE 3.12.0 | 17/Oct/20  | 15/Jan/21    |
++---------------+------------+--------------+
+| OP-TEE 3.11.0 | 22/Aug/20  | 16/Oct/20    |
++---------------+------------+--------------+
+| OP-TEE 3.10.0 | 23/May/20  | 21/Aug/20    |
++---------------+------------+--------------+
```

Comment:
I don't think the start date is helpful, I suggest keeping only the release date.